go.mod

```diff
@@ -1,36 +1,38 @@
 module knative.dev/func
 
-go 1.21
+go 1.22
+
+replace github.com/openshift/source-to-image => github.com/matejvasek/source-to-image v1.3.2-0.20240604071927-a5ace19e3f71
```

Comment:
Can we wait for the `source-to-image` PR is merged if it makes sense timewise?

Reply:
fixed